Competition2017/src/competition/subsystems/autonomous/DriveToHopperCommandGroup.java

```diff
@@ -25,26 +34,39 @@ public DriveToHopperCommandGroup(XPropertyManager propManager,
         distanceFromTurningPointToHopper = propManager
                 .createPersistentProperty("Distance From Turning Point On Baseline To Hopper In Inches", 43.25);
         headingToFaceHopper = propManager.createPersistentProperty("Heading To Face Hopper In Degrees", 90);
+        redAllianceHeadingToFaceHopper = propManager.createPersistentProperty("Red alliance heading to face hopper", 0);
+        blueAllianceHeadingToFaceHopper = propManager.createPersistentProperty("Blue alliance heading to face hopper", 180);
         
         // start with the back wheels both on the wall with the left wheel on the very edge of the wall and move to
         // turning point on baseline (does not move past baseline)
-        DriveForDistanceCommand moveToTurningPointCommand = driveForDistanceProvider.get();
+        moveToTurningPointCommand = driveForDistanceProvider.get();
         moveToTurningPointCommand.setDeltaDistance(distanceFromWallToBaseline);
         this.addSequential(moveToTurningPointCommand);
 
         // change direction to face hopper
-        RotateToHeadingCommand faceHopperCommand = rotateToHeadingProvider.get();
-        faceHopperCommand.setTargetHeadingProp(headingToFaceHopper);
+        faceHopperCommand = rotateToHeadingProvider.get();
```

Comment:
If you don't need multiple instances of a command, you can get an instance injected directly.